### ISSUE TITLE
Pressing lock,  unlocks both doors & charge port at the same time

### ISSUE THREAD

**Issue by Panthium** [Kemmey/Kemmey-TeslaWatch-Public#85]
When I press the lock symbol, both the car doors and the charge port are opening.
Tesla Model S P90DL facelift 2016

**Comment by Kemmey** [comment]
@Panthium Yes - this is by design. I had several users request this feature because they are using non-Tesla cables without release buttons. 

Could you describe how this affects you in a negative way? And under which circumstance. I might be able to fine tune the feature.

**Comment by Panthium** [comment]
Ok but the charge port will open with no charge cable attached and when I'm not at home. For example if I go to my local
Supermarket, I press to unlock the car and both the door unlocks and the charge port opens!

Why would anyone request this feature?

**Comment by Kemmey** [comment]
@Panthium That is definitely a bug - that is not how it's supposed to function. 

A few questions.

1. Does this happen every time?
2. What model and year vehicle do you have? I might need to check vehicle for feature capabilities to eliminate the unwanted opening 
3. If it only happens occasionally, could you perform the following test: before tapping unlock, ensure the titlebar area has a low number - such as 1s to 5s, indicating the watch received new data from the car. My initial suspicion is that the app will have old data that tells it a cable is connected, and so will both unlock and open - but that is just my initial hunch. I think I have a solution for this scenario that I worked on just last night, because the same issue was present when unlocking with Siri - but 100% reproducible every time. 

Cheers,

Kim

**Comment by Panthium** [comment]
Hi, sorry for the slow response.

Car is facelift Model S P90DL 2016 
Watch is series 6

Issue happened yesterday while at B&Q DIY store (charge port opened when unlocking the car), but I didn't check the timer. It's possible if I only use the watch app then it's ok, but if I mix using the fob and app the app gets out of sync. 
I notice that if I unlock the car with app and walk away from the vehicle (car has auto lock enabled) the app still thinks the car is open.


Anyway, let me do some proper testing and I will get back to you.
